# HG changeset patch
# User Rik <rik@octave.org>
# Date 1454166858 28800
#      Sat Jan 30 07:14:18 2016 -0800
# Node ID 36b27fac6bc90310289ffe377643cfa442b2fbb4
# Parent  9c61ab1f758891248612b8c96ecf405f54a4c984
build: Silence warnings about bootclasspath when compiling java scripts.

* scripts/java/module.mk: Add -Xlint:-options to invocation of javac.

diff --git a/scripts/java/module.mk b/scripts/java/module.mk
--- a/scripts/java/module.mk
+++ b/scripts/java/module.mk
@@ -48,17 +48,17 @@ scripts_java_JAVA_CLASSES = $(addprefix 
 scripts_java_JAVA_IMAGES = $(addprefix scripts/java/, $(JAVA_IMAGES))
 
 scripts_DISTCLEANFILES += $(addprefix $(abs_top_builddir), $(java_JAVA_IMAGES))
 
 srcdir_scripts_java_JAVA_IMAGES = $(addprefix $(srcdir)/scripts/java/, $(JAVA_IMAGES))
 
 $(scripts_java_JAVA_CLASSES) : %.class : %.java | scripts/java/$(octave_dirstamp)
 	$(AM_V_GEN)$(MKDIR_P) scripts/java/$(org_octave_dir) && \
-	( cd $(srcdir)/scripts/java; "$(JAVAC)" -source 1.3 -target 1.3 -d $(abs_top_builddir)/scripts/java $(org_octave_dir)/$(<F) )
+		( cd $(srcdir)/scripts/java; "$(JAVAC)" -source 1.3 -target 1.3 -Xlint:-options -d $(abs_top_builddir)/scripts/java $(org_octave_dir)/$(<F) )
 
 scripts/java/.images-stamp: $(srcdir_scripts_java_JAVA_IMAGES)
 	$(AM_V_GEN)if [ "x$(srcdir)" != "x." ]; then \
 	  $(MKDIR_P) scripts/java/$(org_octave_dir)/images; \
 	  cp $(srcdir_scripts_java_JAVA_IMAGES) scripts/java/$(org_octave_dir)/images; \
 	fi && \
 	touch $@
 
